tests/unit/test_objector.py

```diff
@@ -26,8 +26,7 @@ def test_parse_error(self):
         assert isinstance(error, APIException)
 
         error_response = {"json": {"errors": []}}
-        with pytest.raises(ClientException):
-            objector.parse_error(error_response)
+        assert objector.parse_error(error_response) is None
```

Comment:
Do we have any examples where this actually happens?

Reply:
Yes, there are plenty. See other comment.